From ee014b442eac3e85178c89d04691f4fc73cf89e4 Mon Sep 17 00:00:00 2001
From: "Chris \"Koying\" Browet" <cbro@semperpax.com>
Date: Sun, 18 Oct 2015 11:24:00 +0200
Subject: [PATCH] FIX: Only handle 3D bitmap subs in TAB

3D bitmap subs cannot be detected in SBS, and there is always the option
to play with Kodi 3D disabled
---
 xbmc/cores/dvdplayer/DVDCodecs/Overlay/DVDOverlayCodecFFmpeg.cpp | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/xbmc/cores/dvdplayer/DVDCodecs/Overlay/DVDOverlayCodecFFmpeg.cpp b/xbmc/cores/dvdplayer/DVDCodecs/Overlay/DVDOverlayCodecFFmpeg.cpp
index dca4bdf..c9a0008 100644
--- a/xbmc/cores/dvdplayer/DVDCodecs/Overlay/DVDOverlayCodecFFmpeg.cpp
+++ b/xbmc/cores/dvdplayer/DVDCodecs/Overlay/DVDOverlayCodecFFmpeg.cpp
@@ -257,18 +257,13 @@ CDVDOverlay* CDVDOverlayCodecFFmpeg::GetOverlay()
     }
 
     RENDER_STEREO_MODE render_stereo_mode = g_graphicsContext.GetStereoMode();
-    if (render_stereo_mode != RENDER_STEREO_MODE_OFF)
+    if (render_stereo_mode == RENDER_STEREO_MODE_SPLIT_HORIZONTAL)
     {
       if (rect.h > m_height / 2)
       {
         m_height /= 2;
         rect.h /= 2;
       }
-      else if (rect.w > m_width / 2)
-      {
-        m_width /= 2;
-        rect.w /= 2;
-      }
     }
 
     CDVDOverlayImage* overlay = new CDVDOverlayImage();
